lib/sqlalchemy/dialects/postgresql/json.py

```diff
@@ -340,4 +363,30 @@ def contained_by(self, other):
                 CONTAINED_BY, other, result_type=sqltypes.Boolean
             )
 
+        def deletes_field_or_array_element(self, array):
+            """ JSONB expression. Deletes field or array element specified in
+            the argument array.
+            """
+            return self.operate(
```

Comment:
@CaselIT . That's the line I need your opinion.
Should we be defensive here and always convert to `sqlalchemy.dialects.postgresql.array` or leave it to the user ? I'm always inclined to the first option but that's a matter of taste/culture of the codebase, which I'm not familiar with.

Reply:
I'm not sure about thins. I think it should be possible for an user to pass a simply a list like [1, 'b'] and this should work correctly.

We probably need a round trip tests for this. 

Thoughs @zzzeek ?